src/vec_runtime_DYN.c

```diff
@@ -0,0 +1,210 @@
+ /*
+ Copyright (c) [2019] Steven Munroe.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+ vec_runtime_DYN.c
+
+ Contributors:
+      Steven Munroe
+      Created on: Sep 17, 2019
+ */
+
+/*!
+ * \file  vec_runtime_DYN.c
+ * \brief Source file for generating the dynamic IFUNC resolves for
+ * Power Vector Library.
+ *
+ * The working assumption the base and highest supported platform
```

Comment:
The working assumption _is that_ the base...

Reply:
Ok